learners-guild/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Thanks, @deadlyicon. Having echo logging to explicitly mark the beginning of each phase of the script just makes me feel warm and fuzzy, but extra flags might be useful.

Wouldn't `+e` turn the stop-on-error flag off?